src/pivot.coffee

```diff
@@ -421,10 +421,15 @@ callWithJQuery ($) ->
         #`order`: Array of "+" or "-" values, one per attribute. "-"
         #  indicates a descending sort.
         arrSort: (attrs, order) =>
+            # Convert empty value string back to null so it may be compared naturally to other values.
+            _getKeyVal = (key, attrIdx) =>
```

Comment:
Couldn't you just simplify this to:
 ```
convertBlankToNull = (val) => return if keyVal == @emptyVal then null else keyVal)
```
Then line 432 would become
```
 sorter(convertBlankToNull(keyA[attrIdx]), convertBlankToNull(keyB[attrIdx]))
```
I think that function name is more indicative of what's happening and a simpler function at that.